README.md

```diff
@@ -79,6 +79,11 @@ Once the containers are up and running you can copy the source code of the base
 
 **Notice:** Tags/Releases in this GitHub repo are the equivalent to tags in the Docker Cloud repository, master is the **latest** image in DockerHub.
 
+### Stop the containers
+ +1. To stop a running container we can perform the next command in the terminal  `docker stop container name` adding the corresponding container name , in this case with our corresponding container name `docker stop pwr2docker_web_1` .
+ +2. if we have the container running in the foreground we can close it performing `ctrl + c` twice.
```

Comment:
Hi @nanogrunge Thank you for your PR!
Just a comment:  if you "perform" CTRL + C twice you will kill abruptly the container instead of letting it gracefully stop. It is useful to let the process shutdown smoothly to avoid any random issue (corrupted files, hanged sockets, etc). Sometimes it is also useful to forcibly kill a process when it is ignoring a SIGTERM and is running forever, but that is what a single `CTRL + C` (or `docker stop`) does after 10 seconds. 

Conclusion: better perform `CTRL + C` once.